v1/app/app.py

```diff
@@ -45,6 +45,30 @@ def text2img(text: str):
 
     return StreamingResponse(io.BytesIO(img.tobytes()), media_type="image/png")
 
+@app.post("/img2img")
+def img2img(prompt: str, img: UploadFile = File(...)):
+
+   device = get_device()
+   img2img_pipe = StableDiffusionImg2ImgPipeline.from_pretrained("../model")
+   img2img_pipe.to(device)
+
+   img = Image.open(img.file).convert("RGB")
+
+   init_img = img.resize((768, 512))
+
+   # generate an image
+
+   img = img2img_pipe(prompt, init_img).images[0]
+   img = np.array(img)
```

Comment:
Missing the conversion of the image to BGR format from RGB
```suggestion
   dsjdaskdasdnlasndlas
```